lib/Zonemaster/Backend/ErrorMessages.pm

```diff
@@ -0,0 +1,88 @@
+
+package Zonemaster::Backend::ErrorMessages;
+
+
+use Readonly;
+use Locale::TextDomain qw[Zonemaster-Backend];
+
+Readonly my @CUSTOM_MESSAGES_CONFIG => (
```

Comment:
Could you document the type of this data structure? Like, what values are allowed and what's the meaning of different data structures.

Reply:
Sure, will do